buildozer/targets/android.py

```diff
@@ -931,6 +931,17 @@ def execute_build_package(self, build_cmd):
                 asset_dest = asset
             cmd.append(realpath(expanduser(asset_src)) + ':' + asset_dest)
 
+        # support for res folder
+        resources = self.buildozer.config.getlist('app', 'android.add_resources', [])
+        for resource in resources:
+            cmd.append('--add-resource')
+            if ':' in resource:
+                resource_src, resource_dest = resource.split(":")
+            else:
+                resource_src = resource
+                resource_dest = ""
+            cmd.append(realpath(expanduser(resource_src)) + ':' + resource_dest)
```

Comment:
Ouch, duplicating such logic on `buildozer` may lead to something unexpected (E.g: I needed to recall how the no `:` syntax worked on `python-for-android` side.)

This is bad, but I'm NOT blaming you, since we're expanding paths on `buildozer` side everywhere. ( Bad, and should be moved on `python-for-android` ? )

Reply:
Yes, absolutely, but as you point out this is consistent with for example https://github.com/kivy/buildozer/blob/master/buildozer/targets/android.py#L931L938

I choose to copy what is shown to work, it is the low risk implementation. And yes the whole interface needs a rewrite, and an automatic appclean, and the .spec reorganized into common and uncommon, and....